cpp/basix/cell.h

```diff
@@ -35,6 +35,12 @@ xt::xtensor<double, 2> geometry(cell::type celltype);
 /// @return List of topology (vertex indices) for each dimension (0..tdim)
 std::vector<std::vector<std::vector<int>>> topology(cell::type celltype);
 
+/// Get the numbers of entitys connected to each subentity of the cell
+/// @param celltype Cell Type
+/// @return List of topology (vertex indices) for each dimension (0..tdim)
+std::vector<std::vector<std::vector<std::vector<int>>>>
```

Comment:
Could you expand on the doc for the return object? It has four indices - what is each one?

Reply:
I've added to to doc